src/__tests__/TabWorkflow.cy.ts

```diff
@@ -0,0 +1,76 @@
+import schema from '../../cypress/fixtures/workflowSchema.json';
+import { mount } from 'cypress/vue';
+import { defineComponent, useAttrs } from 'vue';
+import TabWorkflow from '@components/dumb/TabWorkflow.vue';
+
+const data = [
+  {
+    actor: 'https://dev-id.erfgoed.net/actoren/12653',
+    datum: '2024-01-09T10:39:15.996776+01:00',
+    state: {
+      id: 0,
+    },
+    owners: [
+      {
+        id: 'https://dev-id.erfgoed.net/actoren/12564',
+        type: 'actor',
+        omschrijving: 'Van Humbeeck, Astrid',
+      },
+    ],
+    actor_omschrijving: 'Proces energieadvies',
+  },
+  {
+    actor: 'https://dev-id.erfgoed.net/actoren/12618',
+    datum: '2024-01-09T15:51:43.786767+01:00',
+    state: {
+      id: 1,
+      comment: 'Test',
+      state_qualifier: 1,
+    },
+    owners: [
+      {
+        id: 'https://dev-id.erfgoed.net/actoren/12564',
+        type: 'actor',
+        omschrijving: 'Van Humbeeck, Astrid',
+      },
+    ],
+    actor_omschrijving: 'Adriaens, Wouter',
+  },
+];
+
+const TestComponent = defineComponent({
+  components: { TabWorkflow },
+  setup() {
+    const attrs = useAttrs();
+    return { attrs };
+  },
+  template: `<div style="width: 100%; height: 300px">
+  <TabWorkflow v-bind="attrs"></TabWorkflow>
+  </div>`,
+});
+
+describe('TabWorkflow - no data', () => {
+  it('renders', () => {
+    mount(TestComponent, { props: { data: [], schema: [] } });
+  });
+
+  it('shows the correct no rows melding', () => {
+    mount(TestComponent, { props: { data: [], schema: [] } });
+    cy.dataCy('ag-overlay').invoke('text').should('contain', 'Er zijn nog geen workflows beschikbaar');
```

Comment:
> Er zijn nog geen workflows beschikbaar

deze zin klopt niet. Beter:
- Er zijn nog geen workflow gegevens beschikbaar
- Er zijn nog geen gegevens beschikbaar
- Er is nog geen workflow beschikbaar
...